test/utils/update/beneficial_owners_managing_officers_data_fetch.spec.ts

```diff
@@ -26,12 +29,14 @@ describe("util beneficial owners managing officers data fetch", () => {
     mockLoggerError.mockReset();
     mockGetCompanyPscService.mockReset();
     mockGetCompanyOfficers.mockReset();
+    mockGetManagingOfficerPrivateData.mockReset();
   });
 
   test("retrieveBoAndMoData sets BO & MO data in appData.update, sets appData.update.bo_mo_data_fetched", async () => {
     appData = {};
     mockGetCompanyPscService.mockReturnValue(MOCK_GET_COMPANY_PSC_ALL_BO_TYPES);
     mockGetCompanyOfficers.mockReturnValue(MOCK_GET_COMPANY_OFFICERS);
+    mockGetManagingOfficerPrivateData.mockReturnValue(MOCK_MO_PRIVATE_DATA);
```

Comment:
Are these changes necessary? We don't call the private data service in `beneficial_owners_managing_officers_data_fetch.ts`